docker/Dockerfile.helper

```diff
@@ -0,0 +1,25 @@
+# =============
+# This file is automatically generated from the templates in stackabletech/operator-templating
+# DON'T MANUALLY EDIT THIS FILE
+# =============
+FROM docker.stackable.tech/stackable/ubi8-rust-builder AS builder
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal AS operator
+LABEL maintainer="Stackable GmbH"
+
+# Update image
+RUN microdnf update --disablerepo=* --enablerepo=ubi-8-baseos --enablerepo=ubi-8-baseos -y \
+  && rm -rf /var/cache/yum \
+  && microdnf install --disablerepo=* --enablerepo=ubi-8-baseos shadow-utils -y \
+  && rm -rf /var/cache/yum
+
+COPY --from=builder /app/stackable-opa-bundle-helper /
+
+RUN groupadd -g 1000 stackable && adduser -u 1000 -g stackable -c 'Stackable OPA Bundle Helper' stackable
+
+USER 1000:1000
+
+ENV WATCH_NAMESPACE="default"
```

Comment:
Not sure i like this (i know there is no setting namespaces so far in opa). But id rather throw the error instead of defaulting to default?

Reply:
removed